src/navsat_transform.cpp

```diff
@@ -100,6 +100,7 @@ NavSatTransform::NavSatTransform(const rclcpp::NodeOptions & options) :
   frequency = this->declare_parameter("frequency", frequency);
   delay = this->declare_parameter("delay", delay);
   transform_timeout = this->declare_parameter("transform_timeout", transform_timeout);
+  queue_size_ = this->declare_parameter("queue_size", 5);
```

Comment:
Ahhhh just when I thought we were done ;-)

Default in ROS1 was `1`, can you make this here too? Also making the queue_size more specific.

... Or just make 1 hardcoded in them and call this done like they were before. If you read the code 1 is the only reasonable number for these anyhow. I'd actually recommend this. No need to add a literally useless parameterization

Reply:
I'll just hard-code 1 for those.

Reply:
Thanks!